docs/presentations/articles/hugging_face.md

```diff
@@ -0,0 +1,101 @@
+# Hugging Face
+
+_Author: [Luis Nothvogel](mailto:luis.nothvogel@htwg-konstanz.de)_  
+  
+## TL;DR  
+  
+Hugging Face has emerged as a pivotal player in the AI and machine learning arena, specializing in natural language processing (NLP). This article delves into its core offerings, including model hosting, spaces, datasets, pricing, and the Terraformer API. Hugging Face is not only a repository for cutting-edge models but also a platform for collaboration and innovation in AI.  
+
+### Model Hosting on Hugging Face
+
+Hugging Face has made a name for itself in model hosting. It offers a vast repository of pre-trained models, primarily focused on NLP tasks such as text classification, question answering, and language generation. According to them they host over 350k models. Users can easily download and deploy these models. Moreover, Hugging Face allows users to upload and share their models with the community.
+
+```python
+from transformers import pipeline, set_seed
+
+# Example of using a pre-trained model
+>>> generator = pipeline('text-generation', model='gpt2')  
+>>> set_seed(42)  
+>>> generated_texts = generator("The student worked on", max_length=30, num_return_sequences=2)  
+>>> print(generated_texts)
```

Comment:
```suggestion
# Example of using a pre-trained model
generator = pipeline('text-generation', model='gpt2')  
set_seed(42)  
generated_texts = generator("The student worked on", max_length=30, num_return_sequences=2)  
print(generated_texts)
```

Like this you _do not_ need to indicate a REPL (because you provide the output as a separate code box). It's just that the code reflects working Python code. Sorry for the confusion ✌🏼